scripts/travis/apt-install

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e -x
+
+install_retry_counter=0
+max_apt_retries=5
+
+# This function loops a couple of times over apt-get, hoping to
+# avoid CI errors due to errors during apt-get
+# hashsum mismatches, DNS errors and similar things
+while true; do
+	let install_retry_counter+=1
+	if [ $install_retry_counter -gt $max_apt_retries ]; then
+		exit 1
+	fi
+	apt-get clean -qqy || continue
+	apt-get update -qqy || continue
+	apt-get install -qqy --no-install-recommends $@ && break
+
+	# In case it is a network error let's wait a bit.
+	sleep $install_retry_counter
```

Comment:
Actually, maybe print something here `Retrying attempt ${install_retry_counter}`?
Probably will make logs easier to understand.

Reply:
Good idea. I added that.